setup.cfg

```diff
@@ -10,7 +10,7 @@ include_trailing_comma = true
 [metadata]
 name = graphkb
 url = https://github.com/bcgsc/pori_graphkb_python
-version = 1.6.0
+version = 1.7.0
```

Comment:
Shouldn't we merge back the master branch into develop instead?

Reply:
We cannot merge master into develop as is, because the tests must pass to allow the merge.

Reply:
Ok, I see.